### PR TITLE
[AKS] Support use custom kubelet identity

### PR DESCRIPTION
**Pull request by norshtein**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
linter_exclusions.yml approved. Hi @fengzhou-msft, please review code, thanks.

**Comment by norshtein** [comment]
@fengzhou-msft I've removed the release information from this PR, will add it to a separate release PR. Could you please help review the code?

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -4076,3 +4099,25 @@ def aks_pod_identity_exception_update(cmd, client, resource_group_name, cluster_
 def aks_pod_identity_exception_list(cmd, client, resource_group_name, cluster_name):
     instance = client.get(resource_group_name, cluster_name)
     return _remove_nulls([instance])[0]
+
+
+def _ensure_cluster_identity_permission_on_kubelet_identity(cli_ctx, cluster_identity_object_id, scope):
+    managed_identity_operator_role = 'Managed Identity Operator'
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/custom.py:4105]: define as consts?

**Review by zqingqing1 — APPROVED** [review]
lgtm with nit

File: src/aks-preview/azext_aks_preview/custom.py (hunk 1 above)
Comment (norshtein) [reply src/aks-preview/azext_aks_preview/custom.py]: Done. Thanks for reviewing and reminding!

**Comment by norshtein** [comment]
Hi @zqingqing1 , @fengzhou-msft , could you please help merge the PR? Thanks!

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**